INSTALL.rst

```diff
@@ -234,6 +234,7 @@ HDF5
   - ``make``
   - *optional:* ``make test``
   - ``make install``
+  - If you encounter errors related to linking MPI during ``./configure``, you might try setting the compiler manually via ``./configure --enable-parallel --enable-shared --prefix $HOME/lib/hdf5/  CC=mpicc CXX=mpic++``.
```

Comment:
there is an unnecessary double whitespace before CC.

note: when testing, double check `CC` is not `export`-ed from a previous session to the same value :)

Reply:
fixed white space

Reply:
Install and most tests went well. 
Both are not exported to the previous session (still `gcc` and `g++`).

Reply:
thanks for testing!